packages/website/data/sponsors.json

```diff
@@ -295,6 +295,14 @@
     "totalDonations": 1500,
     "website": "https://www.mysportsinjury.co.uk"
   },
+  {
+    "id": "420HUBS.COM",
+    "image": "https://images.opencollective.com/420-hubs/0302e85/logo.png",
+    "name": "420HUBS.COM",
+    "tier": "supporter",
+    "totalDonations": 1260,
```

Comment:
Is this correct, @JoshuaKGoldberg 

<img width="160" alt="image" src="https://user-images.githubusercontent.com/7462525/169858348-86d40ae1-3af0-46dc-bcca-9ceb254efe18.png">

On opencollective it looks like they've only donated $13 and they're only a "Supporter" ($3/month)

But on the preview it shows us listing them under the "Gold Supporters" tier:

<img width="487" alt="image" src="https://user-images.githubusercontent.com/7462525/169858821-ac59ef50-be4a-4bb5-af5a-45a3d49d0027.png">

Reply:
Whoever is managing their donations is going on brand here. Monthly donations for $4.20 each over the last three months.

![Screenshot of a Transactions search showing three $4.20 monthly contributions from 420HUBS.COM to typescript-eslint](https://user-images.githubusercontent.com/3335181/169867736-8eeda5e9-f173-4225-99ac-912bca9d9e2d.png)

Here's the data we get back from OpenCollective on them:

```json
{
  account: {
    id: 'gm9bnk80-437xqryw-7w5pvzeo-ljdayw5r',
    imageUrl: 'https://images.opencollective.com/420-hubs/0302e85/logo.png',
    name: '420HUBS.COM',
    website: 'https://420hubs.com'
  },
  tier: {
    amount: { valueInCents: 500 },
    orders: [{ nodes: [ ... ] }]
  },
  totalDonations: { valueInCents: 1260 },
  updatedAt: '2022-03-23T03:16:27.440Z'
}
```

The `orders.nodes` array is size 24. 9 of them have `"status": "CANCELLED"`. That's good evidence that they are (or at least previously were) spam trying to game the system.

Takeaway: I'll add a filter that if there are >5 orders and >=1/3 of them are cancelled, we assume the site is spam.

Reply:
Actually, following up, this would exclude quite a few legitimate donators. Never mind this math. I'll keep digging and send a PR.

Reply:
really - is it that common that people's donations are cancelled? That's so suprising!!

Reply:
#5055